contributors/devel/owners.md

```diff
@@ -69,46 +72,46 @@ aliases:
 
 GitHub usernames and aliases listed in OWNERS files are case-insensitive.
 
-## Code Review Process
+## Code Review using OWNERS files
 
 This is a simplified description of our [full PR testing and merge
 workflow](/contributors/devel/pull-requests.md#the-testing-and-merge-workflow)
 that conveniently forgets about the existence of tests, to focus solely on the roles driven by
 OWNERS files.
 
+### The Code Review Process
+
 - The **author** submits a PR
-- Phase 0: Automation determines **reviewers** and **approvers** for the PR
+- Phase 0: Automation suggests **reviewers** and **approvers** for the PR
   - Determine the set of OWNERS files nearest to the code being changed
-  - Choose two **reviewers**, and assign them to the PR (choose more than one to avoid assigning to
-    an inactive / unavailable reviewer)
+  - Choose two **reviewers**, and request their reviews on the PR (choose more than one to avoid
```

Comment:
We choose *at least* 2 reviewers. Since we start by trying to find a unique reviewer for every relevant leaf OWNERS file we frequently end up requesting reviews from more than the minimum.